packages/fxa-settings/src/components/Settings/ConnectedServices/Service.tsx

```diff
@@ -30,11 +30,11 @@ export function Service({
 }: {
   name: string;
   deviceType: string | null;
-  location: DeviceLocation;
+  location?: DeviceLocation;
   lastAccessTimeFormatted: string;
   handleSignOut: () => void;
 }) {
-  const { city, stateCode, country } = location;
+  const { city, stateCode, country } = location || {};
```

Comment:
I made this edit because I was getting a very red and in your face error message after I tweaked Settings tests. I think I'll take another look at this though - I thought not all of our service mocks had a `location` property but I think they do.